src/include/plan_node/abstract_scan_plan_node.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <memory>
+#include <utility>
+#include "parser/expression/abstract_expression.h"
+#include "plan_node/abstract_plan_node.h"
+
+namespace terrier::plan_node {
+class AbstractScanPlanNode : public AbstractPlanNode {
+ public:
+  AbstractScanPlanNode(std::shared_ptr<OutputSchema> output_schema, parser::AbstractExpression *predicate,
+                       bool is_for_update = false, bool parallel = false)
```

Comment:
probably don't want default values here.

Reply:
Moved the default values to the member variables on the builder template for AbstractScanPlanNode